sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -575,6 +575,24 @@ class JoinSuite extends QueryTest with SharedSQLContext {
         Row(3, 2) :: Nil)
   }
 
+  test("predicate push down in left anti join") {
+    val df = sql("SELECT l.a, l.b FROM testData2 l LEFT ANTI JOIN testData r ON l.a = l.b")
```

Comment:
If you add that file into `spark/sql/core/src/test/resources/sql-tests/inputs`, the result can be generated by the following. Please refer `SQLQueryTestSuite.scala`.
```
SPARK_GENERATE_GOLDEN_FILES=1 build/sbt "sql/test-only *SQLQueryTestSuite"
```
And you can run the test by the following.
```
build/sbt "sql/test-only *SQLQueryTestSuite"
```